### ISSUE TITLE
Unknown TLS backend passed to `use_preconfigured_tls`

### ISSUE THREAD

**Issue by Ziedelth** [crunchy-labs/crunchy-cli#437]
**Describe the bug**
When I try to use a command who need credentials :
```
thread 'main' panicked at crunchy-cli-core/src/lib.rs:402:34:
called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Builder, source: "Unknown TLS backend passed to `use_preconfigured_tls`" }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**To Reproduce**
Steps / command to reproduce the behavior:

Dockerfile
```Dockerfile
FROM rust as rust

# Install ffmpeg
RUN apt-get update && apt-get install -y ffmpeg

RUN git clone https://github.com/crunchy-labs/crunchy-cli.git
WORKDIR /crunchy-cli
RUN cargo build --release
RUN cargo install --force --path .

RUN mkdir /crunchy-cli/data
WORKDIR /crunchy-cli/data

ENTRYPOINT ["crunchy-cli"]
```

```
$ docker run -it  --mount type=bind,source="$(pwd)",target=/crunchy-cli/data --rm --name lytecrh lytecrh:latest --credentials "username:password" download -o DAL_S5EP08.mp4 -a ja-JP -s fr-FR https://www.crunchyroll.com/fr/watch/G9DUE0Z35/the-one-who-pulled-the-trigger
```

**Comment by benkrej** [comment]
Hi, what was the solution to this issue?

**Comment by benkrej** [comment]
Fixed after aligning  `rustls` version to `reqwest` internal `rustls` version

**Comment by enterprisey** [comment]
I have the same error even on the latest git version (commit 756022b955633c73c2920091b3f02d37ab791004). I have verified that the versions specified in the fix commit (commit 509683d23a5689dc958a79440227585c9d383c30) have been downloaded (tokio 1.38 and not so native tls c7ac566).

**Comment by enterprisey** [comment]
Update, only the version installed with `cargo install --git https://github.com/crunchy-labs/crunchy-cli` crashed like this; checking out the repo myself and running it worked fine. So maybe the issue's been fixed.
